jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBDatabaseMetadata.java

```diff
@@ -1674,6 +1509,11 @@ public ResultSet getPrimaryKeys(String catalog, String schema, String table) thr
             TSDataType.TEXT,
             TSDataType.INT32,
             TSDataType.TEXT);
+
+    String sg = "";
```

Comment:
never call it `sg`, we've already renamed it to `db` in new version.

Reply:
check it in the following code.

Reply:
the change has done.